examples/jsm/nodes/accessors/MaterialNode.js

```diff
@@ -201,59 +194,40 @@ class MaterialNode extends Node {
 
 			}
 
-		} else if ( scope === MaterialNode.ROTATION ) {
-
-			node = this.getFloat( 'rotation' );
-
-		} else if ( scope === MaterialNode.UV ) {
+		} else if ( scope === MaterialNode.SHEEN ) {
 
-			// uv repeat and offset setting priorities
+			const sheenNode = this.getColor( 'sheenColor' ).mul( this.getFloat( 'sheen' ) ); // Move this mul() to CPU
 
-			let uvScaleMap =
-				material.map ||
-				material.specularMap ||
-				material.displacementMap ||
-				material.normalMap ||
-				material.bumpMap ||
-				material.roughnessMap ||
-				material.metalnessMap ||
-				material.alphaMap ||
-				material.emissiveMap ||
-				material.clearcoatMap ||
-				material.clearcoatNormalMap ||
-				material.clearcoatRoughnessMap ||
-				material.iridescenceMap ||
-				material.iridescenceThicknessMap ||
-				material.specularIntensityMap ||
-				material.specularColorMap ||
-				material.transmissionMap ||
-				material.thicknessMap ||
-				material.sheenColorMap ||
-				material.sheenRoughnessMap;
+			if ( material.sheenColorMap && material.sheenColorMap.isTexture === true ) {
 
-			if ( uvScaleMap ) {
+				node = sheenNode.mul( this.getTexture( 'sheenColorMap' ).rgb );
 
-				// backwards compatibility
-				if ( uvScaleMap.isWebGLRenderTarget ) {
+			} else {
 
-					uvScaleMap = uvScaleMap.texture;
+				node = sheenNode;
 
-				}
+			}
 
-				if ( uvScaleMap.matrixAutoUpdate === true ) {
+		} else if ( scope === MaterialNode.SHEEN_ROUGHNESS ) {
 
-					uvScaleMap.updateMatrix();
+			const sheenRoughnessNode = this.getFloat( 'sheenRoughness' );
 
-				}
+			if ( material.sheenRoughnessMap && material.sheenRoughnessMap.isTexture === true ) {
 
-				node = uniform( uvScaleMap.matrix ).mul( vec3( uv(), 1 ) );
+				node = sheenRoughnessNode.mul( this.getTexture( 'sheenRoughnessMap' ).a );
 
 			} else {
 
-				node = uv();
+				node = sheenRoughnessNode;
 
 			}
 
+			node = node.clamp( 0.07, 1.0 );
```

Comment:
Why the minimum is 0.7?

Reply:
I think that `0.07` is to prevent incompatible values.
https://github.com/mrdoob/three.js/blob/cdfefb4d3e10a4a4e7212cd7666fd99cd56ac801/src/renderers/shaders/ShaderChunk/lights_physical_fragment.glsl.js#L111